examples/objectServerExample/src/main/java/io/realm/examples/objectserver/CounterActivity.java

```diff
@@ -45,36 +47,35 @@
 import me.zhanghai.android.materialprogressbar.MaterialProgressBar;
 
 public class CounterActivity extends AppCompatActivity {
-
     private static final String REALM_URL = "realm://" + BuildConfig.OBJECT_SERVER_IP + ":9080/~/default";
 
-    private Realm realm;
-    private SyncSession session;
-    private CRDTCounter counter;
-    private SyncUser user;
-    private AtomicBoolean downloadingChanges = new AtomicBoolean(false);
-    private AtomicBoolean uploadingChanges = new AtomicBoolean(false);
-    private ProgressListener downloadListener = new ProgressListener() {
+    private final ProgressListener downloadListener = new ProgressListener() {
         @Override
-        public void onChange(Progress progress) {
+        public void onChange(@Nonnull Progress progress) {
```

Comment:
Isn't it non-null by default?

Reply:
Interesting huh.  No, it doesn't appear to be.  This is not is a package that we've NonNullByDefault-ed.  If this is example code, I think it makes sense to make it look like user code would look and I think it would have these annotations.  I'd be happy to do it by package, if you think that's more typical...

Reply:
Strange, I guess there is something about these annotations I don't understand then. Because the `ProgressListener` interface is in a package covered by a package level annotation. It seems really odd if people implementing this interface in whatever package doesn't inherit that :(